### PR TITLE
Meenu/feat: new derived fx first commit

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/markets/cryptocurrencies/index.tsx [hunk 1]
```diff
@@ -16,7 +16,10 @@ const CryptocurrenciesPage = () => (
             )}
             title={localize('Cryptocurrency market | Crypto trading | Deriv')}
         />
-        <Hero />
+        <DerivedFXHero
+            title="Derived"
+            description="Trade on asset prices derived from real-world or simulated markets. Manage your exposure by selecting the volatility level to suit your risk appetite. Choose from our 24/7 synthetic indices, derived FX indices, and basket indices."
```
Comment (habib-deriv) [inline src/pages/markets/cryptocurrencies/index.tsx:21]: Localize missing

File: src/pages/markets/derived-fx/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const DerivedFxPage = () => (
+    <Layout>
+        <SEO
+            description={localize(
+                'Trade on asset prices derived from real-world or simulated markets. Manage your exposure by selecting the volatility level to suit your risk appetite. Choose from our 24/7 synthetic indices, derived FX indices, and basket indices.',
+            )}
+            title={localize('Derived Fx | Derived Fx demo account | Deriv')}
+        />
+        <DerivedFXHero
+            title="Derived"
```
Comment (habib-deriv) [inline src/pages/markets/derived-fx/index.tsx:19]: same

**Review by habib-deriv — DISMISSED** [review]
LGTM

File: src/components/layout/nav/components/markets/nav-market-mobile.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            >
+                {title}
+            </StyledLinkMarket>
+        </NavLink>
+    )
+}
+
+const NavMarketMobile = () => {
+    return (
+        <MobileWrapper>
```
Comment (prince-deriv) [inline src/components/layout/nav/components/markets/nav-market-mobile.tsx:50]: If the purpose of this wrapper is to hide this on desktop then we have `<Mobile>` component on `visibility.tsx` we can use that instead

File: src/pages/markets/components/sections/_other-markets.tsx [hunk 4]
```diff
@@ -23,11 +21,11 @@ type MarketType = {
 }
 type MarketsType = {
     forex: MarketType
-    synthetic_indices: MarketType
+    //synthetic_indices: MarketType
```
Comment (prince-deriv) [inline src/pages/markets/components/sections/_other-markets.tsx:24]: Let's remove if not needed

File: src/pages/markets/components/sections/_other-markets.tsx [hunk 5]
```diff
@@ -23,11 +21,11 @@ type MarketType = {
...
     stock_indices: MarketType
     commodities: MarketType
     cryptocurrencies: MarketType
-    basket_indices: MarketType
+    //basket_indices: MarketType
```
Comment (prince-deriv) [inline src/pages/markets/components/sections/_other-markets.tsx:28]: Let's remove if not needed

File: src/pages/markets/components/sections/_other-markets.tsx [hunk 6]
```diff
@@ -265,25 +264,34 @@ const MobileCardContainer = styled(Flex)`
...
+`
 
 const OtherMarkets = ({ except }: OtherMarketsProps) => {
     const { is_uk, is_eu } = useCountryRule()
 
     const markets = [
         '',
         'forex',
-        'synthetic_indices',
+        //'synthetic_indices',
```
Comment (prince-deriv) [inline src/pages/markets/components/sections/_other-markets.tsx:283]: Let's remove if not needed

File: src/pages/markets/components/sections/_other-markets.tsx [hunk 7]
```diff
@@ -265,25 +264,34 @@ const MobileCardContainer = styled(Flex)`
...
         'stock_indices',
         'cryptocurrencies',
-        'basket_indices',
+        //'basket_indices',
```
Comment (prince-deriv) [inline src/pages/markets/components/sections/_other-markets.tsx:286]: Let's remove if not needed

File: src/pages/markets/components/sections/_other-markets.tsx [hunk 8]
```diff
@@ -265,25 +264,34 @@ const MobileCardContainer = styled(Flex)`
...
         'commodities',
         '',
     ]
 
     const eu_markets = [
         '',
         'forex',
-        'synthetic_indices',
+        //'synthetic_indices',
```
Comment (prince-deriv) [inline src/pages/markets/components/sections/_other-markets.tsx:294]: Let's remove if not needed

File: src/pages/markets/components/sections/_why-trade.tsx [hunk 9]
```diff
@@ -43,53 +41,66 @@ const ItemContainer = styled(Box)`
...
-        font-size: 1.75rem;
-        padding: 1.25rem 4.75rem;
+        font-size: 14px;
     }
 `
 
-export const WhyTrade = ({ children, header, text }: WhyTradeProps) => {
-    const handleSignup = useHandleSignup()
+export const WhyTrade = ({ children, header, description }: WhyTradeProps) => {
+    // const handleSignup = useHandleSignup()
```
Comment (prince-deriv) [inline src/pages/markets/components/sections/_why-trade.tsx]: Let's entirely remove this

File: src/components/layout/nav/components/markets/nav-market-mobile.tsx (hunk 3 above)
Comment (meenakshi-deriv) [reply src/components/layout/nav/components/markets/nav-market-mobile.tsx]: Hi @prince-deriv , its a styled component for mobile wrapper.

File: src/pages/markets/components/sections/_why-trade.tsx (hunk 9 above)
Comment (meenakshi-deriv) [reply src/pages/markets/components/sections/_why-trade.tsx]: Hi @prince-deriv , removed comments

**Pull request closed without merging**